src/Sylius/Bundle/PromotionBundle/Form/Type/CatalogRule/VariantRuleConfigurationType.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
Should we add here a docblock?

Reply:
we dont have it everywhere, idk :/

Reply:
@GSadee maybe - wdyt?

Reply:
We should have them everywhere. Please add